docs: add gateway timeout and update shared gpu (#5069)
jina-ai/jina: docs/fundamentals/jcloud/advanced.md

    uses: jinahub+docker://Executor1
```

### Timeout

> By default, Jcloud gateway will close connections that idle over `600` seconds. If you want longer connection timeout threshold, you can consider to change the `timeout` parameter in `gateway` tab.

Reviewer: ```suggestion
By default, JCloud gateway will close connections that have been idle for over `600` seconds. If you want longer connection timeout threshold, you can consider changing the `timeout` parameter in `gateway`.
```
